### ISSUE TITLE
code type is case sensitive, ```Java``` can not render java style

### ISSUE THREAD

**Issue by xwjie** [vuejs/vuepress#347]
in marddown file, when insert code, the code type is case sensitive, `Java` can not render with java style.

```Java
// in github , ```Java``` can render with java style, but vuepress can not, must be ```java```
int i = 0;
```

**Comment by xwjie** [comment]
```XML``` also can not render correct

for example:

```xml
<dependency>
	<groupId>org.springframework.boot</groupId>
	<artifactId>spring-boot-starter-security</artifactId>
</dependency>
```

**Comment by ulivz** [comment]
Fixed at 5e87b65

**Comment by ulivz** [comment]
Will release it at 0.9.0: #419. 🚀
